lib/cloudregister/smt.py

```diff
@@ -250,10 +254,8 @@ def __is_cert_valid(self, cert):
     def __request_cert(self):
         """Request the cert from the SMT server and return the request"""
         cert_res = None
-        attempts = 0
         retries = 3
-        while attempts < retries:
-            attempts += 1
+        for attempts in range(retries):
```

Comment:
What does changing the loop construct have to do with the fix?

Reply:
I will revert that change

Reply:
Removed that commit with that change, tests added